### PR TITLE
1.0: in_tail: Add read_bytes_limit_per_second description

### PR DESCRIPTION
**Pull request by cosmo0920**
Documentation for https://github.com/fluent/fluentd/pull/3185.

Signed-off-by: Hiroshi Hatake <hatake@clear-code.com>

### CODE REVIEW

**Comment by ashie** [comment]
Could you resolve the conflict?

**Comment by cosmo0920** [comment]
I'd resolved the conflict but this parameter is not supported for now.
We should merge this PR after https://github.com/fluent/fluentd/pull/3185 is merged.

**Review by ashie — CHANGES_REQUESTED** [review]
* the range of `read_bytes_limit_per_second` should be described (>= 8192)
* `max_thread_pool_size` should be added too

File: input/tail.md [hunk 1]
```diff
@@ -218,6 +218,18 @@ The number of lines to read with each I/O operation.
...
+
+| type    | default        | version |
+|:--------|:---------------|:--------|
+| size    | -1 (unlimited) | 1.12.0  |
+
+The number of reading bytes per second to read with I/O operation.
+
+If you work with a big cluster with high volume of log, you can use this parameter
+to avoid network saturation and make it easier to calculate the max throughput per node.
+This option is useful. To restrict shipping log volumes per second, set a positive number.
```
Comment (ashie) [inline input/tail.md:231]: > `This option is useful`

This sentence isn't needed.

Comment (cosmo0920) [reply input/tail.md]: Fixed in https://github.com/fluent/fluentd-docs-gitbook/pull/259/commits/08df02e330b8406f7332e7bfb307facfda7b7687.

File: input/tail.md [hunk 2]
```diff
@@ -218,6 +218,20 @@ The number of lines to read with each I/O operation.
...
 
+### `read_bytes_limit_per_second`
+
+| type    | default        | version |
+|:--------|:---------------|:--------|
+| size    | -1 (unlimited) | 1.13.0  |
+
+The number of reading bytes per second to read with I/O operation.
+
+This value should be greater than 8192.
```
Comment (ashie) [inline input/tail.md:229]:
```diff
- greater than
+ equal or greater than
```

**Comment by ashie** [comment]
I'll merge it after v1.13 is released.

**Pull request merged**